creation/web_base/logging_utils.source

```diff
@@ -75,7 +77,7 @@ generate_glidein_metadata_json() {
 }
 
 
-log_init_tokens() {
+_glog_log_init_tokens() {
```

Comment:
Should this be `_glog_init_tokens()`? 

If not, then lines 89, 94, and 99 should say `_glog_log_init_tokens`, right?

Reply:
is `_glog_init_tokens`, fixed